README.md

```diff
@@ -13,17 +13,17 @@ repo.
 Please make a pull request or create an issue if you want to contribute.
 
 ### Prerequisites
-* Nodejs version 16+
-* Bootstrap
-* ReactJS
-* AWS S3 bucket config.json
-* ExpressJS
-* Typescript - Superset of JavaScript which primarily provides optional static
-  typing, classes and interfaces.
-* Styled Components - Style components directly using template literals.
 
-### Development
+-   Nodejs version 16+
+-   Bootstrap
+-   ReactJS
+-   AWS S3 bucket config.json
+-   ExpressJS
+-   Typescript - Superset of JavaScript which primarily provides optional static
+    typing, classes and interfaces.
+-   Styled Components - Style components directly using template literals.
 
+### Development
```

Comment:
Prettier changed this. Hope that is ok.

Reply:
Yes as long as you add a prettier github action PR next. So that the repo overall is consistent. Otherwise it doesn't make sense to have random prettier changes throughout this repo's changes

Reply:
Cool, I really like that so I'll add it next